internal/planner/planner.go

```diff
@@ -375,6 +381,8 @@ func (p *Planner) planQueries() error {
 		for _, qv := range qvs.Sorted() {
 			qv = rewrittenVar(qs.RewrittenVars, qv)
 			if !qv.IsGenerated() && !qv.IsWildcard() {
+				// NOTE(sr): We have no location for these: they could appear in multiple
```

Comment:
This is probably fine. These statements are unlikely to be interesting in terms of error messaging, profiling, etc.